scripts/validate-list.js

```diff
@@ -0,0 +1,36 @@
+import Ajv from "ajv";
+import addFormats from "ajv-formats";
+import fs from "node:fs";
+import { exit } from "node:process";
+
+// eslint fails to parse "with { type: "json" }"
+// See https://github.com/eslint/eslint/discussions/15305
+const tokenList = JSON.parse(fs.readFileSync("./src/hemi.tokenlist.json"));
+
+const schemaUrl =
+  "https://raw.githubusercontent.com/Uniswap/token-lists/main/src/tokenlist.schema.json";
+
+// See https://github.com/uniswap/token-lists?tab=readme-ov-file#validating-token-lists
+async function validate() {
+  const ajv = new Ajv({ allErrors: true, verbose: true });
+  addFormats(ajv);
+  const schema = await fetch(schemaUrl).then((r) => r.json());
+  const validator = ajv.compile(schema);
+  const valid = validator(tokenList);
+  if (valid) {
+    return;
+  }
+  if (validator.errors) {
+    throw validator.errors.map(function (error) {
+      delete error.data;
```

Comment:
Does `error.data` contain specifics about why the validation failed? In that case we may want to keep it.

Reply:
This is an example of a failed job, it has enough specifics. See [the failed job](https://github.com/hemilabs/token-list/actions/runs/12915031776/job/36015931845#step:4:6)

<img width="867" alt="image" src="https://github.com/user-attachments/assets/6ed3caa9-bea7-4643-bf73-54c739f30eeb" />

The `delete error.data` comes from the [example reference from Uniswap](https://github.com/uniswap/token-lists?tab=readme-ov-file#validating-token-lists).

I didn't check what's in there, but IMHO the information from the output is more than enough.

Reply:
I just checked and yes, it may be indeed valuable, as it shows the value that failed 😅

Reply:
Updated here [1b2905f](https://github.com/hemilabs/token-list/pull/30/commits/1b2905f14d5c654210a58625815ddb4ba2177adb)

Example with keeping data:

![image](https://github.com/user-attachments/assets/ed481f8f-b349-4d97-ab6a-a71a82a678df)

Although without data was enough, it would require a bit of extra work to find which token failed. This way works better